src/components/Reactions/getUniqueEmojiCodes.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'underscore';
+import getPreferredEmojiCode from './getPreferredEmojiCode';
+
+/**
+ * Given an emoji object and a list of senders it will return an
+ * array of emoji codes, that represents all used variations of the
+ * emoji.
+ * @param {{ name: string, code: string, types: string[] }} emoji
+ * @param {Array} users
+ * @return {string[]}
+ * */
+export default function getUniqueEmojiCodes(emoji, users) {
```

Comment:
Could you please move this to the `lib` folder since it's not a component?

Reply:
There is a `ReportUtils.js` file that it could maybe go in, but that file is becoming a big monster that I'm hesitant to put more stuff in if we can avoid it.

Reply:
<img width="456" alt="Screenshot 2023-05-18 at 23 37 00" src="https://github.com/Expensify/App/assets/129500732/d4b1aa39-6d29-4ccd-a79b-4302719d7af3">

We also are putting getPreferredEmojiCode function in here. So I suggest that we should move 2 functions to `ReportActionUtils` file or `EmojiUtils` file

Reply:
Just updated